app/assets/stylesheets/application.scss

```diff
@@ -1,4 +1,5 @@
 @import "govuk-frontend/govuk/all";
+@import "./govspeak.scss";
 
 /*
 https://github.com/alphagov/govuk_publishing_components/blob/main/docs/install-and-use.md
```

Comment:
@mo-zag it should be possible to just include all the styling from either the `govspeak` or `govuk_publishing_components` gems without needing to copy and maintain them ourselves. Any idea how to get vendor assets into our pipeline?